crates/control/src/behavior/node.rs

```diff
@@ -70,6 +70,7 @@ pub struct CycleContext {
     intercept_ball_parameters: Parameter<InterceptBallParameters, "behavior.intercept_ball">,
     maximum_step_size: Parameter<Step, "step_planner.max_step_size">,
     striker_set_position: Parameter<Point2<Field>, "behavior.role_positions.striker_set_position">,
+    enable_pose_detection: Parameter<bool, "object_detection.object_detection_top.enable">,
```

Comment:
Suggestion for a more fitting name?
```suggestion
    enable_pose_detection: Parameter<bool, "pose_detection.enable">,
```

Reply:
Done